src/mlpack/methods/cf/svd_wrapper_impl.hpp

```diff
@@ -9,8 +9,20 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
+
+#ifndef MLPACK_METHODS_SVDWRAPPER_IMPL_HPP
+#define MLPACK_METHODS_SVDWRAPPER_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include "svd_wrapper.hpp"
+
+namespace mlpack
+{
+namespace cf
+{
```

Comment:
This and the change in svd_wrapper is probably the biggest change made in this PR. Commenting to draw attention.